.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -43,9 +43,8 @@ assignees: ''
     `make -C install/kubernetes/ check-docker-images`
 - [ ] Get the image digests from the build process and make a commit and PR with
       these digests.
-  - [ ] Run `contrib/release/pull-docker-manifests.sh` to fetch the image
-        digests, use the URL of the GitHub run here.
-  - [ ] Create a commit and push as a new PR.
+  - [ ] Run `contrib/release/post-release.sh` to fetch the image
+        digests and submit a PR to update these, use the URL of the GitHub run here.
```

Comment:
I read the following a few lines later, which probably needs to be updated?
```markdown
  - [ ] Copy the text from `digest-vX.Y.Z.txt` (previously generated with
        `contrib/release/pull-docker-manifests.sh`) to the end of the release.
```

After that (and your doc change) I can find no other mention of `contrib/release/pull-docker-manifests.sh` in the repo, should it be deleted?

Reply:
It's used from the new script.

We could probably have some discussion at some point about the various scripts we have under `contrib/release` and whether to better organize them to reduce boilerplate / CLI validation stuff, or even potentially reimplement a more comprehensive system in a better language like Go but for now I've taken the approach of "incrementally build more automation on top of existing scripts" and test the newer changes on each release. In cases like this it means we would retain scripts like `pull-docker-manifests.sh` and just call it from the new one.

Reply:
Ah right, I forgot it was called from the new script, apologies.

Still it could be worth to update the line I quoted above? Simply reading through the steps, it's not clear where `pull-docker-manifests.sh` is called.

Reply:
Ah yes, I didn't parse your comment as two separate asks. I've updated the reference later in this file now. :+1: